src/Workspaces/CSharpTest/Formatting/FormattingTriviaTests.cs

```diff
@@ -1650,20 +1650,21 @@ static void Main(string[] args)
 
         [WorkItem(542546, "http://vstfdevdiv:8080/DevDiv2/DevDiv/_workitems/edit/542546")]
         [WorkItem(44423, "https://github.com/dotnet/roslyn/issues/44423")]
-        [Fact(Skip = "https://github.com/dotnet/roslyn/issues/44423"), Trait(Traits.Feature, Traits.Features.Formatting)]
+        [Fact, Trait(Traits.Feature, Traits.Features.Formatting)]
         public async Task FormatInvalidCode_1()
         {
+            var expected = @"> Roslyn.Utilities.dll!   Basic";
             var content = @">	Roslyn.Utilities.dll! 	Basic";
-            await AssertFormatAsync(content, content);
+            await AssertFormatAsync(expected, content);
         }
 
         [WorkItem(542546, "http://vstfdevdiv:8080/DevDiv2/DevDiv/_workitems/edit/542546")]
         [WorkItem(44423, "https://github.com/dotnet/roslyn/issues/44423")]
-        [Fact(Skip = "https://github.com/dotnet/roslyn/issues/44423"), Trait(Traits.Feature, Traits.Features.Formatting)]
+        [Fact, Trait(Traits.Feature, Traits.Features.Formatting)]
         public async Task FormatInvalidCode_2()
         {
             var content = @">	Roslyn.Utilities.dll! Line 43 + 0x5 bytes	Basic";
-            var expectedContent = @">	Roslyn.Utilities.dll! Line 43 + 0x5 bytes Basic";
+            var expectedContent = @"> Roslyn.Utilities.dll! Line 43 + 0x5 bytes Basic";
```

Comment:
que?

Reply:
`content` contains a hard tab. Previously the formatter didn't touch this like (likely treated as skipped trivia), but now the tree shape changed and the formatter converted the tab to a single space character following the `<` operator. I didn't investigate the details because it didn't seem problematic.

Reply:
wfm